recipes/rtmidi/4.x.x/conandata.yml

```diff
@@ -1,7 +1,7 @@
 sources:
   "4.0.0":
-    url: "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-4.0.0.tar.gz"
-    sha256: "370cfe710f43fbeba8d2b8c8bc310f314338c519c2cf2865e2d2737b251526cd"
+    url: "https://github.com/thestk/rtmidi/archive/refs/tags/4.0.0.zip"
```

Comment:
This is an auto-generated source file archive, the current download is the official download, so I think we should stick with it.

Reply:
Done here: b41ce0e